### PR TITLE
v1.6.0 - Added star ratings and selectable star ratings

### PR DESCRIPTION
**Pull request by tommaton**
### Added
- Star ratings and selectable star ratings documentation

### CODE REVIEW

**Comment by DamianMullins** [comment]
The files under the `/templates` directory should have been ignored by git, you don't need to check them in.

File: assets/src/scss/_dependencies.scss [hunk 1]
```diff
@@ -29,7 +28,7 @@
  * be accessed via mixins
  *
  */
-
+@import 'settings/variables';
```
Comment (ashleynolan) [inline assets/src/scss/_dependencies.scss]: Was this just to make sure the asset path resolves correctly?

Comment (tommaton) [reply assets/src/scss/_dependencies.scss]: yep.

Comment (DamianMullins) [reply assets/src/scss/_dependencies.scss]: Should it go under a new heading? Doesn't seem like it should live under a "Fozzie Extensions" heading?

Comment (ashleynolan) [reply assets/src/scss/_dependencies.scss]: Yeh - it should do.  I think the comment above should be associated with the `card` include below, and then this dependency should have it's our section above this comment but below the `f-icons` include.

**Pull request merged**